test/e2e/components/vl-cookie-consent.js

```diff
@@ -29,8 +39,9 @@ class VlCookieConsent extends VlElement {
         return Promise.all(optIns.map(optIn => new VlCookieConsentOptIn(this.driver, optIn)));
     }
 
-    async bewaarKeuze() {
-        await (await this._getModal()).submit();
+    async save() {
+        const modal = await this._getModal();
+        await modal.submit();
```

Comment:
breaking change!

Reply:
Zeker, dus rekening mee te houden tijdens release.

Reply:
wel mee te pakken om af en toe te overwegen of een rename van nederlands naar engels wel een breaking change waard is